sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -558,21 +557,26 @@ public void onCallFailed(Exception e) {
         if (isInvalidPartitionWhenDocuments(partition, result)) {
             return result;
         }
-        getTokenAndCallCosmosDbApi(
-                partition,
-                result,
-                new TokenExchangeServiceCallback(mTokenManager) {
+        postAsyncGetter(new Runnable() {
+            @Override
```

Comment:
Nit: add a new line break before `@Override` - our conventions it to have an empty line before Java annotation.
```suggestion

            @Override
```